steps/pipeline-tests-jest.yaml

```diff
@@ -0,0 +1,52 @@
+parameters:
+- name: environment
+  default: ''
+- name: test_file
+  default: ''
+- name: workingDirectory
+  default: ''
+- name: junit_output_dir
+  default: ''
+- name: test_title
+  default: ''
+- name: dns_zone
+  default: platform.hmcts.net
+- name: product
+  default: ''
+
+steps:
+- checkout: self
+- checkout: cnp-azuredevops-libraries
+
+- task: Bash@3
+  displayName: Get dns zone names
+  inputs:
+    filePath: $(System.DefaultWorkingDirectory)/cnp-azuredevops-libraries/scripts/get-dns-zone-names.sh
+  env:
+    ENVIRONMENT: ${{ parameters.environment }}
+    PRODUCT: ${{ parameters.product }}
+
+- task: Bash@3
+  displayName: Run pipeline tests
+  inputs:
+    targetType: 'inline'
+    script: |
+      mkdir -p ${{ parameters.junit_output_dir }}
+      yarn
+      yarn test-junit
```

Comment:
there a reason it's using test-junit instead of the standard `test`?

Reply:
There are no output files when junit isn't used and the tests don't appear under the tests tab in Azure DevOps as a result

Reply:
that's to do with the test task though? shouldn't have anything to do with the name here.

basically the standard task is `test` bit confused why it's being overridden

Reply:
Found the culprit https://github.com/hmcts/azure-platform-terraform/pull/1231/files/f02253216555b47c0da566412c5299ab8d4ba962..7ef37df2952cd6cf47e1f81cbf5d2afc8788a1ce